### ISSUE TITLE
[Qt] Transmission not have icon Open URL for Toolbar

### ISSUE THREAD

**Issue by varlesh** [transmission/transmission#814]
Qt UI looked as unfinished without **Open URL** icon: [cencored]

- Ubuntu 18.04
- DE: Plasma 5
- Transmission 2.92 (14714)

**Comment by mikedld** [comment]
Already fixed in c77946e49a49965a149233c1cd45ae9e89e66f87 (pending 3.00 release). But if you have better ideas for the fix please do tell.

**Comment by varlesh** [comment]
@mikedld  It's strange used similar icons for different actions....
```
ui.action_AddURL->setIcon (getStockIcon (QLatin1String ("document-open"), QStyle::SP_DialogOpenButton,
                                           QStringList () << QLatin1String ("emblem-web") << QLatin1String ("applications-internet")));
```
`document-open` used for `ui.action_OpenFile`. I think it's will be looking not good - icons will be duplicated and this spoil style.

Also:
`emblem-web` - it's emblem
`applications-internet` -  it's aplications/categories icon.
But **Open URL** it's action and will be prefer used icon from **actions** for that.

How about `insert-link` - this icon available on Faenza and Breeze too...
